### PR TITLE
fix(cooldowns): command#data#name and relative timestamp fix

### PR DESCRIPTION
**Pull request by AmNobCop**
Updated how the message to inform the user of the amount of time they need to wait before using the command again looks. Now it looks better with the format of the unix timestamp. Also added more clarity to add the code to the InteractionCreate event, not just anywhere in the main file.
 
@Joeyk710 - Helped with changes

### CODE REVIEW

File: guide/additional-features/cooldowns.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ if (timestamps.has(interaction.user.id)) {
 
 	if (now < expirationTime) {
 		const expiredTimestamp = Math.round(expirationTime / 1000);
-		return interaction.reply({ content: `Please wait <t:${expiredTimestamp}:R> more second(s) before reusing the \`${command.data.name}\` command.`, ephemeral: true });
+		return interaction.reply({ content: `Please wait, you are on a cooldown for \`${command.data.name}\`. You can use it again <t:${expiredTimestamp}:R>`, ephemeral: true });
```
Comment (jaw0r3k) [inline guide/additional-features/cooldowns.md:74]: maybe use [time](https://old.discordjs.dev/#/docs/discord.js/main/function/time) function

File: guide/additional-features/cooldowns.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ client.cooldowns = new Collection();
 
 The key will be the command names, and the values will be Collections associating the user's id (key) to the last time (value) this user used this command. Overall the logical path to get a user's last usage of a command will be `cooldowns > command > user > timestamp`.
 
-In your main file, add the following code:
+In your InteractionCreate event, add the following code:
```
[the remark this answers is not in the document]
Comment (AmNobCop) [reply guide/additional-features/cooldowns.md:33]: I mean it does, but the event is called that, ima check what the guide has it as and go off that.

File: guide/additional-features/cooldowns.md (hunk 1 above)
Comment (AmNobCop) [reply guide/additional-features/cooldowns.md:74]: 🤷 I mean I could, but I see no need to.

**Comment by AmNobCop** [comment]
Combined other pull request

File: code-samples/additional-features/cooldowns/index.js [hunk 3]
```diff
@@ -51,7 +51,7 @@ client.on(Events.InteractionCreate, async interaction => {
 
 		if (now < expirationTime) {
 			const expiredTimestamp = Math.round(expirationTime / 1000);
-			return interaction.reply({ content: `Please wait <t:${expiredTimestamp}:R> more second(s) before reusing the \`${command.name}\` command.`, ephemeral: true });
+			return interaction.reply({ content: `Please wait, you are on a cooldown for \`${command.data.name}\`. You can reuse it again <t:${expiredTimestamp}:R>`, ephemeral: true });
```
Comment (TetieWasTaken) [inline code-samples/additional-features/cooldowns/index.js:54]: I'm not against the `#name` fix. But this semantic change seems unnecessary.

File: guide/additional-features/cooldowns.md (hunk 1 above)
Comment (TetieWasTaken) [inline guide/additional-features/cooldowns.md:74]: This semantic change seems unnecessary.

File: guide/additional-features/cooldowns.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Idris1401) [reply guide/additional-features/cooldowns.md:33]: > I mean it does, but the event is called that, ima check what the guide has it as and go off that.

`Events.InteractionCreate` returns `'interactionCreate`
the event name is "interactionCreate"

[the remark this answers is not in the document]
Comment (jaw0r3k) [reply guide/additional-features/cooldowns.md:33]: however more readable is just `interaction create`

File: guide/additional-features/cooldowns.md (hunk 1 above)
Comment (Jiralite) [reply guide/additional-features/cooldowns.md:74]: The former is unacceptable—the relative timestamp would end up looking like this:

> Please wait `in 30 seconds` more seconds[...]

I believe this current change is fine.

File: code-samples/additional-features/cooldowns/index.js (hunk 3 above)
Comment (Jiralite) [inline code-samples/additional-features/cooldowns/index.js:54]:
```suggestion
			return interaction.reply({ content: `Please wait, you are on a cooldown for \`${command.data.name}\`. You can use it again <t:${expiredTimestamp}:R>`, ephemeral: true });
```

File: guide/additional-features/cooldowns.md (hunk 1 above)
Comment (Jiralite) [inline guide/additional-features/cooldowns.md:74]:
```suggestion
		return interaction.reply({ content: `Please wait, you are on a cooldown for \`${command.data.name}\`. You can use it again <t:${expiredTimestamp}:R>.`, ephemeral: true });
```

**Review by Jiralite — APPROVED** [review]

**Pull request merged**